bazel/ray_deps_setup.bzl

```diff
@@ -94,34 +94,34 @@ def ray_deps_setup():
 
     auto_http_archive(
```

Comment:
Can you add a comment like "These dependency rules are copied to external repositories. All references to build files and patches in the Ray repository should be prefixed with the workspace name @com_github_ray_project_ray, instead of just '//'."

Reply:
> Can you add a comment like "These dependency rules are copied to external repositories. All references to build files and patches in the Ray repository should be prefixed with the workspace name @com_github_ray_project_ray, instead of just '//'."

Nice catch. It's updated in lastest commit.